src/resources/views/crud/fields/date_range.blade.php

```diff
@@ -19,13 +19,14 @@ function formatDate($entry, $dateFieldName)
             return $formattedDate;
         }
     }
+
     if (isset($field['value'])) {
         if (isset($entry) && ! is_array($field['value'])) {
             $start_value = formatDate($entry, $field['name'][0]);
             $end_value = formatDate($entry, $field['name'][1]);
-        } elseif (is_array($field['value'])) {
-            $start_value = $field['value'][$field['name'][0]];
-            $end_value = $field['value'][$field['name'][1]];
+        } elseif (is_array($field['value'])) { // gets here when inside repeatable
+            $start_value = current($field['value']); // first array item
+            $end_value = next($field['value']); // second array item
```

Comment:
@pxpm , notice what I've done here. Used `current()` and `next()` to get the first and second item of the array, no matter what the key is. I think (in the future) we can use this in more places where we're not sure what the key is.